README.md

```diff
@@ -75,7 +75,8 @@ Open in a browser or with a code editor.
 
 ### Deployment
 
-You can deploy this project using a browser.
+This project has been deployed using Github Pages. See website link below:
+https://drew246.github.io/Portfolio-application-deployment/
```

Comment:
Good job using GitHub Pages to deploy web pages. 👍

![image](https://github.com/Drew246/Portfolio-application-deployment/assets/78251927/09659049-4306-46cb-8274-029d1406ae42)

[OPTIONAL] 

- [ ] We can use [basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax), to create sophisticated formatting for our prose and code on GitHub with simple syntax. 🤗

- [ ] We can create an inline [link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links) by wrapping link text in brackets `[ ]`, and then wrapping the URL in parentheses `( )`

E.g
```
See the website link below:
[Live Demo](https://drew246.github.io/Portfolio-application-deployment/)
```


See the website link below:
[Live Demo](https://drew246.github.io/Portfolio-application-deployment/)